source/deprecations/v1.x.html.md

```diff
@@ -1260,6 +1324,18 @@ Ember 1.13 deprecated `bind-attr` in favor of the new syntax.
 
 To aid in the migration, you can use the [legacy-bind-attr plugin](https://github.com/emberjs/legacy-bind-attr) to restore this behavior in Ember 2.0 and silence the deprecation warning.
 
+#### Using `this.get('template')`
+
+Prior to 1.13, you developers had to check for the existance of the internal `template` property to determine the existance of a yielded block.
```

Comment:
sorry, meant to be "your developers", but agreed that just "developers" would be best